### DISCUSSION TITLE
Pyracanny not working

### DISCUSSION THREAD

**Discussion by ahalyabr** [lllyasviel/Fooocus#2568] [Q&A] [answered]
the pose of input prompt not matching output even with Stop At:0.95 and weight:1

**Comment by ILikeToasters** [comment, marked as answer]
Did you try with CPDS? Some images work better with that. Also you can go into the debug settings and under the control tab check off the Debug Preprocessors to see the results of Pyracanny. 
<img width="249" alt="Screenshot 2024-03-19 125246" src="https://github.com/lllyasviel/Fooocus/assets/4357657/04616a5f-5027-457e-ac88-29e08254e942">
<img width="565" alt="Screenshot 2024-03-19 125334" src="https://github.com/lllyasviel/Fooocus/assets/4357657/6ee9b80b-9861-4510-95cf-2ffa2c8ea347">
